paper.md

```diff
@@ -0,0 +1,178 @@
+
+# Instructional design
+This lesson material was designed using the concepts from the Carpentries Curriculum Development Handbook (REFERENCE https://cdh.carpentries.org/).
+Most importantly, we used 'backward design': We started with identifying learning objectives: the core skills and concepts that learners need to acquire as a result of the lesson.
+Then, these learning objectives were used to create exercises that test whether the objectives are met.
+Eventually, the content is written in such a way that it matches what learners need to learn to successfully apply these skills.
+
+Furthermore, the lesson is built around live coding. 
+The lesson is built up of small blocks. In each block first the instructor demonstrates how to do something,
+and students follow along on their own computer. Then, the students work independently on exercises individually
+or in groups to test their skills.
+
+The lesson material is built in the new Carpentries lesson template (REFERENCE). 
+This makes the lesson material both suited as a self-study resource, a syllabus to share with students after the workshop,
+as well as lesson material for the instructor. With the 'instructor view' we even include instructor notes 
+at the level of the lesson content.
+
+The lesson is split up in a general introduction, and 3 episodes that cover 3 distinct increasingly more complex deep learning problems.
+Each of the deep learning problems is approached using the same 10-step deep learning workflow.
+By going through the deep learning cycle 3 times on different problems we hope that after the lesson learners will
+feel confident in applying this deep learning workflow to their own work.
+
+
+# Feedback
+This course was taught 7 times over the course of 3 years, both online and in person, at the Netherlands eScience Center. 
+**@Peter @Colin how often did you teach this at which institutes?**
+Apart from the core group of contributors, the workshop was also taught at 3 independent institutes, namely:
+University of Wisconson-Madison, University of Auckland, and EMBL Heidelberg.
+In general, adoption of the lesson material by the instructors not involved in the project went well.
+
+## Student responses
+The feedback we gathered from students is in general very positive, further confirming our statement of need.
+We use the student's feedback to continuously improve the lesson. 
+Some responses from students to the question 'What was your favourite or most useful part of the workshop. Why?' further support the need for this workshop:
+
+_I enjoyed the live coding and playing with the models to see how it would effect the results. 
+It felt hands on and made it easy for me to understand the concepts._
+
+_Well-defined steps to be followed in training a model is very useful. Examples we worked on are quite nice._
+
+_The doing part, that really helps to get the theory into practice._
```

Comment:
```suggestion
The feedback we gathered from students is in general very positive, 
with some responses from students to the question 'What was your favourite or most useful part of the workshop. Why?' further confirming our statement of need:

> _I enjoyed the live coding and playing with the models to see how it would effect the results. 
> It felt hands on and made it easy for me to understand the concepts._

> _Well-defined steps to be followed in training a model is very useful. Examples we worked on are quite nice._

> _The doing part, that really helps to get the theory into practice._

We use the student's feedback to continuously improve the lesson. 
```

Reorganising slightly.